app/views/jobRespView.html

```diff
@@ -0,0 +1,32 @@
+{% extends "layout_unbranded.html" %} 
+
+{% block content %}
+<a href="/job-roles" class="govuk-back-link">Back</a>
+<br />
+
+{% if errormassege %}
+<div id="error" class="govuk-error-message">{{errormassege}}</div> 
+{% else %}
+
+{% if responsibilities and responsibilities.length > 0 %}
+
+<h2>{{responsibilities[0].role_name}} Responsibilities</h2> 
```

Comment:
I will be leaving a comment on the backend PR which may change how you reference the role_name